### PR TITLE
[ci] feat: use packing in test_model_patches

### PR DESCRIPTION
**Pull request by Coach257**
### What does this PR do?
1. test_model_patches now test a packed sequence alignment with two data samples
2. rm eager as eager do not support packing 

https://github.com/ByteDance-Seed/VeOmni/pull/523#discussion_r2876637292

### CODE REVIEW

File: tests/models/test_models_patch.py [hunk 1]
```diff
@@ -122,6 +134,9 @@ def forward_backward_step(self, state_dict: Dict[str, torch.Tensor], model_mode:
                 dtype=self.model.dtype
             )  # qwen3 omni didn't handle dtype in audio_forward
 
+        if batch["position_ids"].dim() == 3 and batch["position_ids"].shape[1] == 3:
+            batch["position_ids"] = batch["position_ids"].transpose(0, 1).contiguous()
+
```
[the remark this answers is not in the document]
Comment (Coach257) [reply tests/models/test_models_patch.py:139]: pretranspose here as hf native modeling can only receive (3, bs, seqlen)

**Review by Luosuu — APPROVED** [review]
